back/engines/commercial/content_builder/spec/acceptance/homepage_layouts_spec.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'ContentBuilderLayouts' do
+  explanation 'Content builder layouts for projects.'
+
+  before { header 'Content-Type', 'application/json' }
+
+  context 'when not signed in' do
+    let!(:homepage) { create(:home_page) }
+    let(:code) { 'homepage' }
+    let(:layout) { homepage.content_builder_layouts.find_by(code: code) }
+
+    get 'web_api/v1/home_pages/content_builder_layouts/:code' do
+      example_request 'Get one layout by code' do
+        assert_status 200
+
+        json_response = json_parse(response_body)
+        expect(json_response).to include(
```

Comment:
You could use the api helper `json_response_body` here and in other places to compact the tests slightly